www/views/lights.html

```diff
@@ -80,7 +80,25 @@
 								<button id="speedUp" onclick="appLampSpeedUp()">Speed Up</button><button id="speedDown" onclick="appLampSpeedDown()">Speed Down</button><br />
 								<button id="discoUp" onclick="appLampDiscoMode()">Disco Mode</button><button id="speedUpLong" onclick="appLampSpeedUpLong()">Exclude Lamp</button><br />
 							</div>
+							<div id="optionsRGBW" style="display:none;">
```

Comment:
This will show up for all RGBW/RGBWW lights. I will see if I can change this so it's only for Limitless.